src/main/java/com/coveros/selenified/element/check/azzert/AssertEquals.java

```diff
@@ -0,0 +1,278 @@
+     */
+    public void text(String expectedText) {
+        String text = checkText(expectedText, 0, 0);
+        if (text == null) {
+            assertNotNull("No element found", text);
+        }
+        assertEquals("Text Mismatch", expectedText, text);
+    }
+
+    /**
+     * Verifies that the element's text in a particular cell equals the provided
+     * expected text. If the element isn't present, or a table, this will
+     * constitute a failure, same as a mismatch. This information will be logged
+     * and recorded, with a screenshot for traceability and added debugging
+     * support.
+     *
+     * @param row          - the number of the row in the table - note, row numbering
+     *                     starts at 1, NOT 0
+     * @param col          - the number of the column in the table - note, column
+     *                     numbering starts at 1, NOT 0
+     * @param expectedText - what text do we expect to be in the table cell
+     */
+    public void text(int row, int col, String expectedText) {
+        String text = checkText(row, col, expectedText, 0, 0);
+        if (text == null) {
+            String reason = "No element found";
+            if (getElement().is().present()) {
+                reason = "Element not table";
+            }
+            assertNotNull(reason, text);
+        }
+        assertEquals("Text Mismatch", expectedText, text);
+    }
+
+    /**
+     * Verifies that the element's value equals the provided expected value. If
+     * the element isn't present or an input, this will constitute a failure,
+     * same as a mismatch. This information will be logged and recorded, with a
+     * screenshot for traceability and added debugging support.
+     *
+     * @param expectedValue the expected input value of the element
+     */
+    public void value(String expectedValue) {
+        String value = checkValue(expectedValue, 0, 0);
+        if (value == null) {
+            String reason = "No element found";
+            if (getElement().is().present()) {
+                reason = "Element not input";
+            }
+            assertNotNull(reason, value);
+        }
+        assertEquals("Value Mismatch", expectedValue, value);
+    }
+
+    /**
+     * Verifies that the element's selected option equals the provided expected
+     * option. If the element isn't present or a select, this will constitute a
+     * failure, same as a mismatch. This information will be logged and
+     * recorded, with a screenshot for traceability and added debugging support.
+     *
+     * @param expectedText the expected input text of the element
+     */
+    public void selectedOption(String expectedText) {
+        String option = checkSelectedOption(expectedText, 0, 0);
+        if (option == null) {
+            String reason = "No element found";
+            if (getElement().is().present()) {
+                reason = "Element not select";
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Element not select" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1192)